packages/forms/src/__tests__/index.spec.ts

```diff
@@ -3,7 +3,6 @@ import * as api from '..';
 it('exposes public api', () => {
   expect(api).toMatchInlineSnapshot(`
     Object {
-      "EMPTY_ERROR_MESSAGE": "EMPTY_ERROR_MESSAGE",
```

Comment:
@umrzoq-toshkentov We should not remove this constant from public APIs. Can you re export it?

Reply:
Let's include constayfile in index.ts file